docs/installation/mount-tracking.md

```diff
@@ -0,0 +1,60 @@
+# Automatic mount tracking
+
+If you are wanting to use enzyme in a large-scale application, you may want to automate
+calls to `unmount()` via a lifecycle hook in your test environment, rather than at the
+individual test level. Enzyme allows for this as part of its configuration:
+
+ES6:
+```js
+// setup file
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({
+  adapter: new Adapter(),
+  enableMountTracking: true,
```

Comment:
it seems kind of weird to call this option "mount tracking" when it tracks both shallow and mount. maybe "enableSandbox", since `sinon-sandbox` is actually the behavior it's emulating?